lib/internal/errors.js

```diff
@@ -234,3 +259,11 @@ function oneOf(expected, thing) {
     return `of ${thing} ${String(expected)}`;
   }
 }
+
+function bufferOutOfBound(name, isWriting) {
+  if (isWriting) {
+    return 'Attempt to write outside buffer bounds';
+  } else {
+    return `"${name}" is out of bounds`;
```

Comment:
`"${name}" is outside of buffer bounds`
It's just that "out of bounds" without context has a bit of a wider meaning 😄 
![image](https://user-images.githubusercontent.com/96947/27957611-2b227400-62ed-11e7-8b58-08e37b849349.png)